src/store/modules/app.ts

```diff
@@ -200,15 +199,25 @@ const actions = {
         ['erc20BalanceOf', { address: space.address, decimals }]
       ];
       const spaceStrategies = spaces[space.key].strategies || defaultStrategies;
-      const scores: any = await Promise.all(
-        spaceStrategies.map((strategy: any) =>
-          strategies[strategy[0]](rpcProvider, [address], strategy[1], blockTag)
+      const scores: any = (
```

Comment:
It seem that you are linting the repo with different rules than the ones set on this repository, this is changing the code formatting in many place of the repo, you may want to check your setup.